ts/src/kucoin.ts

```diff
@@ -847,32 +1151,27 @@ export default class kucoin extends Exchange {
         return result;
     }
 
-    async fetchTransactionFee (code: string, params = {}) {
```

Comment:
why did u remove the type here?

Reply:
@carlosmiei  sorry, I was away and returned atm,  couldnt reply earlier. so the type removal from method signatures and some other incompleteness were due to the reason that this PR was quite old and I've just migrated them into new kucoin.ts with quick copy-pasting and missed those details sorry :( 
would pay more attention to such cases next times